RealEstate/apps/pending/templates/pending/inviteHomebuyer.html

```diff
@@ -0,0 +1,28 @@
+{% extends "core/base.html" %}
+{% load bootstrap3 %}
+
+{% block title %}
+  Invite Homebuyers
+{% endblock %}
+
+{% block body %}
+  <div class="row">
+    <div class="col-sm-3"></div>
+    <div class="col-sm-6">
+      <h2 style="margin-top:50px">
+        <strong>Invite Homebuyer Couple</strong>
```

Comment:
Yeah, I was confused. I initially thought this was the landing page for a HB after clicking on the email invite link. Disregard.